config/env/default.js

```diff
@@ -28,5 +28,5 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: './modules/core/client/img/brand/favicon.ico'
```

Comment:
@mleanos Here's where else it is used: https://github.com/meanjs/mean/blob/master/modules/core/server/views/layout.server.view.html#L32